docs/admin/install.rst

```diff
@@ -336,9 +336,77 @@ This command will take a considerable amount of time and approximately 4GB of ba
+To start the SecureDrop Client, double-click the SecureDrop desktop icon that was set up by the previous command. The preflight updater will start and check for updates. The system should be up-to-date and no updates should be required, but if updates are available follow the instructions in the preflight updater to apply them.
+
+Once the update check is complete, the SecureDrop Client will launch. Log in using an existing journalist account and verify that sources are listed and submissions can be downloaded, decrypted, and viewed.
+
+Post-Install Configuration
+--------------------------
+Once you have verified that you can log into the SecureDrop Client, you may want to consider making additional configuration changes:
+
+- setting up password management
+- enabling selected clipboard usage, if appropriate
+- enabling copying of system logs to a networked VM.
+
+By default, copy and paste is fully disabled for all SecureDrop Workstation VMs. Copying files from or to SecureDrop VMs using ``qvm-copy`` is also disabled.
+
+.. _Password Management Section:
+
+Password management
+~~~~~~~~~~~~~~~~~~~
+If you have installed Qubes with the default configuration, you should have a VM called ``vault``, which is a networkless VM based on the ``fedora-30`` template. You can use this VM to store SecureDrop login credentials or the passphrases for encrypted export USB drives.
+
+Note that all users of this workstation will have access to the password database. If the workstation will be used by more than one journalist, you may want to encourage the use of a smartphone password manager for managing individual login credentials.
+
+The ``fedora-30`` template includes the KeePassXC password manager. It is not listed in the application menu for ``vault`` by default; you can add it by selecting it from the list of available apps in **Q > Domain: vault > Qube Settings > Applications** and pressing the button labeled **>** (do not press the button labeled **>>**, which will add *all* applications to the menu).
+
+After confirming this change, you can launch KeePassXC from the **Domain: vault** menu. On first start, it will suggest enabling automatic updates, which you should decline: ``vault`` is networkless, so the built-in update check will fail; the app will be updated through system updates instead.
+
+When creating a new password database, KeePassXC will prompt you to protect it with a password, and will refuse to accept a blank password. If you want to configure easy access to the password database, you can protect the database using a key file, via **Add additional protection > Add Key File > Generate** on the final screen. Once you have configured a key file, KeePassXC will accept leaving the master password field blank.
```

Comment:
Do we provide any instructions on how to copy the password itself to this new database?

Reply:
Not currently. May want to test importing an Admin Workstation KeePassXC DB into Qubes ...

Reply:
Added further instructions in cf923f620bbff1eb63cdb3df060356a6b6ea0ba2.

Reply:
Oh, also, because this is in the post-install config, the USB device will be grabbed by `sd-devices` first. Will make a note of that.

Reply:
Clarified `sd-devices` behavior in b07ac1948fe4c804320087525e85901fb183f269.

Reply:
as it's moved to initial install, the sd-devices stuff is moot :)